lib/vdsm/network/link/setup.py

```diff
@@ -55,3 +74,15 @@ def parse_nets_bridge_opts(nets):
                 opt.split('=', 1) for opt in opts_str.split(' ')
             )
             yield name, bridge_opts
+
+
```

Comment:
Eh cannot comment that directly on that code. Please remove `setup_custom_bridge_opts`.